File: java-suite/src/main/java/de/fraunhofer/iem/classesAndObjects/classes/abstractClass/Cat.java
package de.fraunhofer.iem.classesAndObjects.classes.abstractClass;

/**
 * Class that extends Animal
 *
 * @author Ranjith Krishnamurthy
 */
public class Cat extends Animal {
    public Cat(String DNA) {
        super(DNA);
    }
    @Override
    public void makeNoise() {
        System.out.println("Meow");
    }
}

File: kotlin-suite/src/main/kotlin/de/fraunhofer/iem/classesAndObjects/classes/abstractClass/abstractExtendsNonAbstract/Cat.kt
package de.fraunhofer.iem.classesAndObjects.classes.abstractClass.abstractExtendsNonAbstract

/**
 * Class that extends Animal
 *
 * @author Ranjith Krishnamurthy
 */
class Cat(DNA: String) : Animal(DNA) {
    override fun makeNoise() {
        println("Meow")
    }
}